proto/sign/signingMessages.go

```diff
@@ -109,58 +110,77 @@ func (sm *SigningMessage) UnmarshalBinary(data []byte) error {
 	return protobuf.DecodeWithConstructors(data, sm, cons)
 }
 
+type JSONdata struct{
+		Len int64
+		Data []byte
+}
+
```

Comment:
It should be deleted in app/conode/defs/messg as it is not used anymore. Perhaps it could be removed here, too...